2023-09-12 15:36:46,734 INFO    StreamThr :49 [internal.py:wandb_internal():86] W&B internal server running at pid: 49, started at: 2023-09-12 15:36:46.733499
2023-09-12 15:36:46,739 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status
2023-09-12 15:36:46,740 INFO    WriterThread:49 [datastore.py:open_for_write():85] open: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/run-zki67ezo.wandb
2023-09-12 15:36:46,750 DEBUG   SenderThread:49 [sender.py:send():369] send: header
2023-09-12 15:36:46,750 DEBUG   SenderThread:49 [sender.py:send():369] send: run
2023-09-12 15:36:47,983 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: check_version
2023-09-12 15:36:47,986 INFO    SenderThread:49 [dir_watcher.py:__init__():211] watching files in: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files
2023-09-12 15:36:47,986 INFO    SenderThread:49 [sender.py:_start_run_threads():1100] run started: zki67ezo with start time 1694504206.734834
2023-09-12 15:36:47,987 DEBUG   SenderThread:49 [sender.py:send_request():396] send_request: summary_record
2023-09-12 15:36:47,990 INFO    SenderThread:49 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-09-12 15:36:47,992 DEBUG   SenderThread:49 [sender.py:send_request():396] send_request: check_version
2023-09-12 15:36:49,009 INFO    Thread-12 :49 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/wandb-summary.json
2023-09-12 15:36:52,985 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: keepalive
2023-09-12 15:36:52,995 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status_report
2023-09-12 15:36:53,035 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: run_start
2023-09-12 15:36:53,118 DEBUG   HandlerThread:49 [system_info.py:__init__():31] System info init
2023-09-12 15:36:53,118 DEBUG   HandlerThread:49 [system_info.py:__init__():46] System info init done
2023-09-12 15:36:53,118 INFO    HandlerThread:49 [system_monitor.py:start():181] Starting system monitor
2023-09-12 15:36:53,119 INFO    SystemMonitor:49 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-09-12 15:36:53,119 INFO    HandlerThread:49 [system_monitor.py:probe():201] Collecting system info
2023-09-12 15:36:53,121 INFO    SystemMonitor:49 [interfaces.py:start():190] Started cpu monitoring
2023-09-12 15:36:53,122 INFO    SystemMonitor:49 [interfaces.py:start():190] Started disk monitoring
2023-09-12 15:36:53,123 INFO    SystemMonitor:49 [interfaces.py:start():190] Started gpu monitoring
2023-09-12 15:36:53,124 INFO    SystemMonitor:49 [interfaces.py:start():190] Started memory monitoring
2023-09-12 15:36:53,125 INFO    SystemMonitor:49 [interfaces.py:start():190] Started network monitoring
2023-09-12 15:36:53,139 DEBUG   HandlerThread:49 [system_info.py:probe():195] Probing system
2023-09-12 15:36:53,174 DEBUG   HandlerThread:49 [gitlib.py:_init_repo():53] git repository is invalid
2023-09-12 15:36:53,175 DEBUG   HandlerThread:49 [system_info.py:probe():240] Probing system done
2023-09-12 15:36:53,175 DEBUG   HandlerThread:49 [system_monitor.py:probe():210] {'os': 'Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-09-12T07:36:53.139494', 'startedAt': '2023-09-12T07:36:46.701692', 'docker': None, 'cuda': None, 'args': ('wandb', 'login'), 'state': 'running', 'program': '/mnt/d/PROJECTION/TianChi/STS-2D/train.py', 'codePath': 'train.py', 'host': 'DESKTOP-LT0ASMB', 'username': 'zy', 'executable': '/home/zy/anaconda3/envs/kaggle_torch/bin/python3', 'cpu_count': 3, 'cpu_count_logical': 5, 'cpu_freq': {'current': 3192.009, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 3192.009, 'min': 0.0, 'max': 0.0}, {'current': 3192.009, 'min': 0.0, 'max': 0.0}, {'current': 3192.009, 'min': 0.0, 'max': 0.0}, {'current': 3192.009, 'min': 0.0, 'max': 0.0}, {'current': 3192.009, 'min': 0.0, 'max': 0.0}], 'disk': {'total': 250.98248672485352, 'used': 63.274070739746094}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 24.93661880493164}}
2023-09-12 15:36:53,175 INFO    HandlerThread:49 [system_monitor.py:probe():211] Finished collecting system info
2023-09-12 15:36:53,175 INFO    HandlerThread:49 [system_monitor.py:probe():214] Publishing system info
2023-09-12 15:36:53,175 DEBUG   HandlerThread:49 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-09-12 15:36:53,178 DEBUG   HandlerThread:49 [system_info.py:_save_pip():67] Saving pip packages done
2023-09-12 15:36:53,178 DEBUG   HandlerThread:49 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-09-12 15:36:53,391 ERROR   HandlerThread:49 [system_info.py:_save_conda():85] Error saving conda packages: [Errno 2] No such file or directory: 'conda'
Traceback (most recent call last):
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/wandb/sdk/internal/system/system_info.py", line 81, in _save_conda
    subprocess.call(
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda'
2023-09-12 15:36:53,398 DEBUG   HandlerThread:49 [system_info.py:_save_conda():86] Saving conda packages done
2023-09-12 15:36:53,410 INFO    HandlerThread:49 [system_monitor.py:probe():216] Finished publishing system info
2023-09-12 15:36:53,428 DEBUG   SenderThread:49 [sender.py:send():369] send: files
2023-09-12 15:36:53,429 INFO    SenderThread:49 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-09-12 15:36:53,436 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: stop_status
2023-09-12 15:36:53,436 DEBUG   SenderThread:49 [sender.py:send_request():396] send_request: stop_status
2023-09-12 15:36:53,791 DEBUG   SenderThread:49 [sender.py:send():369] send: telemetry
2023-09-12 15:36:54,091 INFO    Thread-12 :49 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/output.log
2023-09-12 15:36:54,091 INFO    Thread-12 :49 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/requirements.txt
2023-09-12 15:36:54,092 INFO    Thread-12 :49 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/wandb-metadata.json
2023-09-12 15:36:54,093 INFO    Thread-12 :49 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/conda-environment.yaml
2023-09-12 15:36:54,926 INFO    wandb-upload_0:49 [upload_job.py:push():133] Uploaded file /tmp/tmp727w0955wandb/wgi2ug25-wandb-metadata.json
2023-09-12 15:36:56,099 INFO    Thread-12 :49 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/output.log
2023-09-12 15:36:58,110 INFO    Thread-12 :49 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230912_153646-zki67ezo/files/output.log
2023-09-12 15:36:58,904 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status_report
2023-09-12 15:37:03,905 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status_report
2023-09-12 15:37:08,436 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: stop_status
2023-09-12 15:37:08,437 DEBUG   SenderThread:49 [sender.py:send_request():396] send_request: stop_status
2023-09-12 15:37:08,919 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status_report
2023-09-12 15:37:13,126 INFO    cpu       :49 [interfaces.py:monitor():140] Process cpu has exited.
2023-09-12 15:37:13,127 DEBUG   SystemMonitor:49 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-09-12 15:37:13,128 DEBUG   SystemMonitor:49 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-09-12 15:37:13,128 DEBUG   SystemMonitor:49 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-09-12 15:37:13,129 DEBUG   SenderThread:49 [sender.py:send():369] send: stats
2023-09-12 15:37:14,039 INFO    MainThread:49 [internal.py:handle_exit():76] Internal process exited
2023-09-12 15:37:14,130 DEBUG   HandlerThread:49 [handler.py:handle_request():144] handle_request: status_report
